components/emoji_picker/emoji_picker.stories.js

```diff
@@ -0,0 +1,136 @@
+import { action } from '@storybook/addon-actions';
+import { createTemplateFromVueFile } from '@/common/storybook_utils';
+import DtEmojiPicker from './emoji_picker.vue';
+import DtEmojiPickerDefaultTemplate from './emoji_picker_default.story.vue';
+import DtEmojiPickerWithPopoverTemplate from './emoji_picker_popover.story.vue';
+
+const recentlyUsedEmojis = [
+  {
+    name: 'thumbs up',
+    category: 'people',
+    shortname: ':thumbsup:',
+    shortname_alternates: [':+1:', ':thumbup:'],
+    keywords: ['+1', 'hand', 'thumb', 'up', 'uc6'],
+    unicode_output: '1f44d',
+    unicode_character: '1f44d',
+  },
+  {
+    name: 'thumbs up: medium-light skin tone',
+    category: 'people',
+    shortname: ':thumbsup_tone2:',
+    shortname_alternates: [':+1_tone2:', ':thumbup_tone2:'],
+    keywords: ['+1', 'hand', 'medium-light skin tone', 'thumb', 'up', 'uc8'],
+    unicode_output: '1f44d-1f3fc',
+    unicode_character: '1f44d-1f3fc',
+  },
+  {
+    name: 'thumbs up: dark skin tone',
+    category: 'people',
+    shortname: ':thumbsup_tone5:',
+    shortname_alternates: [':+1_tone5:', ':thumbup_tone5:'],
+    keywords: ['+1', 'dark skin tone', 'hand', 'thumb', 'up', 'uc8'],
+    unicode_output: '1f44d-1f3ff',
+    unicode_character: '1f44d-1f3ff',
+  },
+  {
+    name: 'person: light skin tone',
+    category: 'people',
+    shortname: ':adult_tone1:',
+    shortname_alternates: [':adult_light_skin_tone:'],
+    keywords: ['gender-neutral', 'light skin tone', 'uc10'],
+    unicode_output: '1f9d1-1f3fb',
+    unicode_character: '1f9d1-1f3fb',
+  },
+  {
+    name: 'woman with veil: dark skin tone',
+    category: 'people',
+    shortname: ':woman_with_veil_tone5:',
+    shortname_alternates: [':woman_with_veil_dark_skin_tone:'],
+    keywords: ['uc13'],
+    unicode_output: '1f470-1f3ff-200d-2640-fe0f',
+    unicode_character: '1f470-1f3ff-2640',
+  },
+];
+
+export const argsData = {
+  onSkinTone: action('skin-tone'),
+  onSelectedEmoji: action('selected-emoji'),
+  onClose: action('close'),
+  tabSetLabels: [
+    'Most recently used',
+    'Smileys and people',
+    'Nature',
+    'Food',
+    'Activity',
+    'Travel',
+    'Objects',
+    'Symbols',
+    'Flags',
+  ],
```

Comment:
Yeah we specifically want the user to pass them in because of i18n. If we had i18n translation in dialtone then it would make sense to set them as default.